.vscode/launch.json

```diff
@@ -26,11 +26,11 @@
       "type": "node",
       "request": "launch",
       "cwd": "${workspaceRoot}",
-      "program": "${workspaceRoot}/packages/salesforcedx-vscode-apex-debugger/out/src/adapter/apexDebug.js",
+      "program": "${workspaceRoot}/packages/salesforcedx-apex-debugger/out/src/adapter/apexDebug.js",
```

Comment:
I haven't seen one that separates the extension from the adapter. They bundle into one and name it like 'vscode-<language>-debug'.